ui/src/components/Header/index.tsx

```diff
@@ -1,78 +1,81 @@
 // import NetworkSelection from "../NetworkSelection"
 
 interface Props {
-  className?: string
+    className?: string
+    handleDrawerOpen: () => void,
 }
 
-const Header = ({ className }: Props) => {
-  const { accountList } = useAccounts()
+const Header = ({className, handleDrawerOpen}: Props) => {
+  const {accountList} = useAccounts()
 
   return (
-    <AppBar className={className}>
-      <Container maxWidth="xl">
-        <Toolbar disableGutters>
-          <Typography
-            variant="h6"
-            noWrap
-            component="div"
-            sx={{ flexGrow: 1 }}
-          >
+    <MuiAppBar position="fixed" className={className}>
+      <Toolbar>
+        <TypographyStyled
+          variant="h6"
+          noWrap
+        >
             Multix
-          </Typography>
-          {!!accountList?.length && (
-            <>
-              <Box className="buttonContainer">
-                <Button
-                  component={Link}
-                  to="/"
-                  className="buttonHeader"
-                >
-                  Home
-                </Button>
-                <Button
-                  component={Link}
-                  to="/create"
-                  className="buttonHeader"
-                >
-                  New Multisig
-                </Button>
-                <Button
-                  component={Link}
-                  to="/about"
-                  className="buttonHeader"
-                >
-                  About
-                </Button>
-                <Button
-                  component={Link}
-                  to="/help"
-                  className="buttonHeader"
-                >
-                  Help
-                </Button>
-              </Box>
-              <MultiProxySelection />
-            </>
-          )}
-          {/* <NetworkSelection /> */}
-        </Toolbar>
-      </Container>
-    </AppBar>
+        </TypographyStyled>
+        {!!accountList?.length && (
+          <BoxStyled>
+            {ROUTES.map(({url, name}) => (
+              <Button
+                component={Link}
+                to={url}
+                className="buttonHeader"
+              >
+                {name}
+              </Button>
+            ))}
+          </BoxStyled>
+        )}
+        <MultiProxySelection/>
+        {/*/!*<NetworkSelection />*!/*/}
+        <IconButtonStyled
+          color="inherit"
+          aria-label="open drawer"
+          edge="end"
+          onClick={handleDrawerOpen}
+        >
+          <MenuIcon/>
+        </IconButtonStyled>
+      </Toolbar>
+    </MuiAppBar>
   )
 }
 
-export default styled(Header)(({ theme }) => `
-  .buttonContainer {
-    flex-grow: 1;
-    display: flex;
-  }
+const BoxStyled = styled(Box)(({theme}) => `
+    display: none;
+    
+    @media (min-width: ${theme.breakpoints.values.sm}px) {
+        flex-grow: 1;
+        display: flex;
+    }
+`)
+
+
+const TypographyStyled = styled(Typography)`
```

Comment:
Can use css classes here? So in the code:
```
<Typography
  className="appTitle"
  variant="h6"
  noWrap
/>
```

and then in the css at the end:
```
  .appTitle {
    flex-grow: 1;
  }
```

Reply:
I think it's up to us to define how we want things to look like. While the way you worked here is better to keep things together (if I remove the Typography, the styling will be removed as well), but it's also more code, and makes you jump more when you read it. I've no strong opinion, and I'm not sure we should enforce something, but the rest of the code definitely has a lot of classes, and almost no styled component.

Reply:
let's keep this as is, and we can go either way, I don't think that's a big deal